build/lily/notifier.env

```diff
@@ -5,16 +5,19 @@ LILY_REPO=/var/lib/lily
 #LILY_CONFIG=/var/lib/lily/config.toml
 
 # Postgres URL may be overridden
-#LILY_STORAGE_POSTGRESQL_DB_URL=postgres://user:pass@host:port/database?options
+LILY_STORAGE_POSTGRESQL_DB_URL="postgres://postgres@timescaledb:5432/postgres"
 
 # Enable IMPORT_SNAPSHOT below to use snapshot on lily startup
 #_LILY_DOCKER_INIT_IMPORT_MAINNET_SNAPSHOT=true
 
 # Debugging options
 #LILY_TRACING=true
-LILY_PROMETHEUS_PORT="prometheus:9090"
+LILY_PROMETHEUS_PORT="localhost:9090"
```

Comment:
im not sure this works, I wasn't able to scrap any metrics from the nodes.

Reply:
I pushed a change if you want to try this again?